.all-contributorsrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "files": ["readme.md"],
+  "imageSize": 100,
+  "contributorsPerLine": 7,
+  "contributorsSortAlphabetically": false,
```

Comment:
By default contributors sort by time they were added, if we want to sort them by name, we can set `contributorsSortAlphabetically` to `true.